sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -250,7 +256,6 @@ static synchronized void unsetInstance() {
      * @param throwable An exception.
      */
     @SuppressWarnings({"SameParameterValue", "WeakerAccess"})
-    // TODO remove the suppress when releasing to jCenter
```

Comment:
I see you remove the suppress on the `trackException` below but leave it here. why?

Reply:
Because we don't use this signature in test app so it will always warn.
The other one, Android Studio is smart enough to detect we use it via reflection so it was warning about redundant suppression.